Create 2020-11-05-recrutement-devs-beta (#4513)
betagouv/beta.gouv.fr: content/_jobs/2020-11-04-recrutement-devs-beta.md

---
roles: 8 développeurs/ses
startup: beta.gouv.fr
open: true
title: Nous recrutons 8 développeurs ou développeuses, juniors et expérimenté.e.s pour rejoindre 8 produits beta.gouv.fr
---

L'incubateur de la DINUM recherche plusieurs développeurs et développeuses juniors et expérimentés pour intégrer différentes équipes et les aider à résoudre un problème de politique publique au contact direct des usagers. 

## Qui sommes-nous ?

> beta.gouv.fr met en œuvre une politique d’innovation publique en créant de petites équipes autonomes et agiles, nommées Startups d’État qui développent des produits numériques en cycles courts, en lien direct avec leurs usagers et les parties prenantes à la politique publiques à laquelle ils contribuent.

Reviewer: ```suggestion
beta.gouv.fr met en œuvre une politique d’innovation publique en créant de petites équipes autonomes et agiles, nommées Startups d’État, qui développent des produits numériques en cycles courts, en lien direct avec leurs usagers et les parties prenantes à la politique publiques à laquelle ils contribuent.
```

> La communauté beta.gouv.fr de plus de 500 personnes, a, depuis la création de la première Startup d'État, décidé collectivement que la meilleure manière de mettre en œuvre une politique publique passe par la constitution d'une équipe autonome. Cette équipe est elle-même constituée de membres issus du public ou du privé qui se donnent pour mission de résoudre les problèmes de ses usagers.

Reviewer: ```suggestion
La communauté beta.gouv.fr de plus de 500 personnes a, depuis la création de la première Startup d'État, décidé collectivement que la meilleure manière de mettre en œuvre une politique publique passe par la constitution d'une équipe autonome. Cette équipe est elle-même constituée de membres issus du public ou du privé qui se donnent pour mission de résoudre les problèmes de ses usagers.
```

1. l'obsession de la satisfaction de nos utilisateurs ;
2. la priorité donnée aux tâches qui ont une utilité réelle ;
3. la certitude qu'il est possible de mettre la technologie au service de l'intérêt général et de la rendre accessible et compréhensible par tous ;
4. l'intuition que la mise en oeuvre d'une politique publique, même numérique, ne peut se faire sans une forte dose de relations humaines ;
5. et enfin la volonté de toujours nous améliorer, d'améliorer le service que nous rendons à nos utilisateurs et de partager au maximum ce que nous apprenons avec les communautés qui nous entourent.

Par notre travail, nous cherchons à montrer que des équipes autonomes, responsables et à qui l'on fait confiance peuvent construire des services numériques qui fonctionnent, simples, faciles à utiliser, et qui améliorent le service public.

Nous documentons régulièrement nos pratiques pour les Startups d’État et pour la communauté sur des sujets aussi divers que nos rituels, nos formations, notre fonds d’accélération des Startups d’État ou encore les outils que nous utilisons. 

Nos statistiques d’impact sont visibles ici : https://beta.gouv.fr/stats

## La mission 

> Depuis 2013, nous avons construits une centaine de produits sur différents sujets en lien avec des incubateurs partenaires de ministères, opérateurs de l’État ou collectivité. 

Reviewer: ```suggestion
Depuis 2013, nous avons construit une centaine de produits sur différents sujets en lien avec des incubateurs partenaires de ministères, opérateurs de l’État ou collectivité. 
```

> Depuis 2013, nous avons construits une centaine de produits sur différents sujets en lien avec des incubateurs partenaires de ministères, opérateurs de l’État ou collectivité. 

Reviewer: Proposition de reformulation (c'est dommage de ne pas glisser "problème" ici), et précision en fin de phrase
```suggestion
Depuis 2013, nous avons construits une centaine de produits répondant à des problèmes variés, en lien avec des incubateurs partenaires, portés par des ministères, opérateurs de l’État ou collectivités.
```

> Que ces produits soient en investigation, en construction ou en accélération, voire en transfert, ils sont tous constitués par une équipe autonome avec un.e coach, un.e intrapreneur.e, un.e ou plusieurs chargé.e.s de déploiement et un ou plusieurs développeurs/ses. 

Reviewer: ```suggestion
Que ces produits soient en investigation, en construction ou en accélération, voire en transfert, ils sont tous constitués par une équipe autonome avec un·e coach, un·e intrapreneur·e, un·e ou plusieurs chargé·e·s de déploiement et un·e ou plusieurs développeu·rs·ses. 
```

Author: Ça me met outdated mais j'ai bien pris en compte

> Que ces produits soient en investigation, en construction ou en accélération, voire en transfert, ils sont tous constitués par une équipe autonome avec un.e coach, un.e intrapreneur.e, un.e ou plusieurs chargé.e.s de déploiement et un ou plusieurs développeurs/ses. 

Reviewer: Suggestion alternative car je trouve la graphie inclusive trop… intrusive:
```suggestion
Que ces produits soient en investigation, en construction ou en accélération, voire en transfert, ils sont tous menés par une équipe autonome dont la composition fait appel à différents métiers, parmi lesquels coaching, intrapreneuriat, déploiement auprès des usagers, et bien sûr développement logiciel.
```

> Que tu sois junior, expérimenté.e ou sénior, ta mission si tu l’acceptes en deux parties :

Reviewer: ```suggestion
Que tu sois junior, expérimenté·e ou sénior, ta mission si tu l’acceptes tient en deux parties :
```

- Mettre en place les bonnes pratiques favorables à la qualité de la conception et du code
- Automatiser les procédures de déploiement et autres outils de productivité
- Veiller à l’optimisation des performances et à la sécurité des solutions

#### La partie transverse : 
Découvrir et contribuer à l’animation de la communauté beta.gouv.fr sur nos formations, la participation à nos décisions collectives et à la vie de l’incubateur, la diffusion de l’approche beta.gouv.fr dans l’administration, la participation aux événements des partenaires de l’incubateur (Etalab, Social Tech, etc) et l’apprentissage puis la transmission des outils internes à l’incubateur (GitHub, Pad, etc.).

## Ton profil :
Les personnes autonomes se plaisent bien dans la communauté beta.gouv.fr.

Une personne autonome est capable de se gérer dans le travail : elle définit ses propres objectifs et trace la voie qui permet de les atteindre.

Si tu n’as pas besoin d’une supervision constante pour avancer sur ce que tu as prévu de faire, tu seras à ton aise ici.

Nous serions très contents de te rencontrer si :

> - Tu es intéressé.e par la résolution de problèmes de politiques publiques ;

Reviewer: ```suggestion
- Tu es intéressé·e par la résolution de problèmes de politiques publiques ;
```

- Tu as un côté entrepreneurial que tu as envie de développer ; 

> - Tu n’as pas de problème à gérer et prioriser des tâches très nombreuses et très différentes ;

Reviewer: Si je suis candidat·e je lis ça comme "c'est le bazar dans le management de la boîte et on va t'exploiter à fond jusqu'au burnout". Red flag. Mieux:

```suggestion
- Tu sais gérer tes priorités parmi des tâches variées, tu sais dire "non" en cas de trop-plein ;
```

Author: c'est pas la vérité ? haha je plaisante.

> - Tu parviens à gérer tes projets de A à Z en autonomie ;

Reviewer: ```suggestion
- Tu sais délimiter des tâches que tu peux mener de A à Z en autonomie, et demander de l'aide si tu les as sous-estimées ;
```

- Tu aimes les modes de communication asynchrones ;
- Tu n’as pas peur d'interagir avec une grande variété d’interlocuteurs ;

> - Tu lances les projets/les choses, même si elles sont imparfaites ;

Reviewer: Pas super clair ? Il me semble que c'est redondant avec le point qui suit, fusionner ?
```suggestion
- Tu n'hésites pas à montrer ou livrer ton travail, même s'il est encore imparfait (bref tu dis souvent «Done is better than perfect »);
```

> - Et tu utilises souvent la phrase «Done is better than perfect » ;

Reviewer: Cette ligne est redondante maintenant

- Tu maîtrises le français et un peu l’anglais dans un contexte professionnel et tu possèdes des qualités rédactionnelles ;
- Tu souhaites travailler au service du bien commun et espères participer à la construction d'une solution qui cherche à tout prix à avoir le maximum d'impact et une utilité avérée ;
- Tu sais travailler en équipe, tu sais écouter et tu acceptes de te remettre parfois en question – ou de remettre en question les procédures adoptées par l'équipe – lorsque tu penses pouvoir faire mieux ;
- Tu as des connaissances en développement logiciel (algorithmique, structuration de données, architecture logicielle, etc.) ;
- Tu as des connaissances de base en intégration et déploiement continu ou encore des connaissances méthodologiques en développement itératif ;
- Tu as une sensibilité UX/accessibilité pour la partie front ;
- Tu as de l’expérience sur un langage de programmation (parmi Python, Javascript (NodeJS ou framework front), Ruby, Java, Scala ou équivalent).

> Et si tu es plus expérimenté.e (+ de 5 ans d’expérience), si :

Reviewer: ```suggestion
Et si tu es plus expérimenté·e (+ de 5 ans d’expérience), si :
```

- Tu as de très bonnes connaissances en développement logiciel ;
- Tu as de solides connaissances en intégration et déploiement continu ;
- Tu peux démontrer tes connaissances en méthodes agiles en environnement devops et lean ;
- Tu sais t’engager en grande responsabilité pour livrer un produit ;
- Tu es fiable et rigoureux dans ton exécution ;
- Tu as une solide culture de la sécurité des systèmes d’information ;
- Tu as de l’expérience dans l’animation et la coordination de travaux en équipe devops et lean ;
- Tu as de l’expérience sur au moins deux langages de programmation ou langages équivalents (parmi Python, Javascript (NodeJS ou framework front), Ruby, Java, Scala) ;
- Et tu comprends les enjeux et de la réglementation dans le domaine de la protection des données personnelles.

> Ne te censures pas si tu ne coches pas toutes les cases ci-dessus et que tu ne connais pas très bien l'écosystème de l’administration mais que tu as envie d’apprendre et de progresser. 

Reviewer: 👏 
```suggestion
Ne te censure pas si tu ne coches pas toutes les cases ci-dessus et que tu ne connais pas très bien l'écosystème de l’administration mais que tu as envie d’apprendre et de progresser. 
```

## Les détails à connaître : 

#### Contrat
Poste ouvert pour un CDD à temps plein, en contrat de vacataire, d’une durée de 6 mois.
Si tu te poses la question, nous sommes presque tous contractuels ou indépendants au sein de la communauté beta.gouv.fr.

#### Rémunération
Pour une développeuse ou un développeur jusqu’à 5 ans d’expérience, le salaire proposé est de 3000 euros nets et pour une développeuse ou un développeur au-delà de 5 ans d’expérience le salaire est de 3500 euros nets. 

#### Travail à distance

> Le travail à distance est possible, ponctuellement, sans qu’un préavis ou une validation ne soient nécessaires — sois simplement raisonnable. 

Reviewer: Je tique sur "ponctuellement" et "raisonnable" ? C'est super vague, et ça me ferait drôle de lire ça alors qu'on est encore sous confinement et qu'on ne sait pas de quoi 2021 sera fait. Je ferais la distinction:
```suggestion
Les équipes sont actuellement organisées pour privilégier le travail à distance, et dans des circonstances plus "normales" il est accepté sous réserve d'une organisation efficace de l'équipe. N'hésite pas à préciser tes souhaits et tes possibilités en la matière.
```

Pour contribuer aux rituels d’équipes et discuter avec les autres membres de beta.gouv.fr, les équipes se croisent à leur rythme parfois un jour par semaine, parfois un jour tous les 15 jours au vu des conditions sanitaires actuelles.  

#### Date de début
Le lundi 4 janvier 2021 

#### Formation

> Nous formons celles et ceux qui rejoignent la communauté beta.gouv.fr aux méthodes des Startups d'État. Un cours en ligne qui reprend les principaux enseignements est à ta disposition ici.

Reviewer: Manque le lien :)

Author: oui haha merci

Au sein des équipes, les membres veilleront à te former sur nos outils si tu ne les connais pas et nous te donnerons la liste des acronymes à connaître pour bien naviguer au sein de l’administration. 

#### Perspectives

> Une grande partie des personnes qui rejoignent la communauté beta.gouv.fr pour une mission de courte période finissent par rester chez nous, parfois pour participer à la croissance de la Startup d'État sur laquelle elles interviennent, parfois pour rejoindre d'autres équipes, parfois pour endosser des rôles plus transverses.

Reviewer: Je trouve que ce paragraphe est trop en mode "promesse" par rapport à ce qui suit. Deux suggestions corrélées:
```suggestion
Une grande partie des personnes qui rejoignent la communauté beta.gouv.fr pour une mission de courte période expriment le souhait de rester chez nous, parfois pour participer à la croissance de la Startup d'État sur laquelle elles interviennent, parfois pour rejoindre d'autres équipes, parfois pour endosser des rôles plus transverses.
```

> Attention, nous ne pouvons pas garantir, même si cela peut être possible, la pérennisation du poste au delà de cette période de 6 mois sous la forme d’un contrat.

Reviewer: ```suggestion
Attention, nous ne pouvons pas garantir la pérennisation du poste au delà de cette période de 6 mois. Nous pouvons cependant te renseigner sur différentes possibilités pour poursuivre une collaboration réussie, par exemple des missions futures en tant que freelance ou en rejoignant l'une des entreprises attributaires de nos marchés, ou une candidature sur un autre type de poste.
```

#### Procédure de recrutement

> Un jury constitué des différentes équipes intéressés par les profils se réunira pour rencontrer les candidats avant de prendre une décision. Si d'aventure cela s'avère compliqué au vu de la situation sanitaire, nous saurons nous adapter.

Reviewer: ```suggestion
Un jury constitué des différentes équipes intéressés par les profils se réunira pour rencontrer les candidat·e·s avant de prendre une décision. Si d'aventure cela s'avère compliqué au vu de la situation sanitaire, nous saurons nous adapter.
```

> Un jury constitué des différentes équipes intéressés par les profils se réunira pour rencontrer les candidats avant de prendre une décision. Si d'aventure cela s'avère compliqué au vu de la situation sanitaire, nous saurons nous adapter.

Reviewer: > Si d'aventure cela s'avère compliqué

C'est pas clair pour moi si je m'imagine candidat, et l'expression peut être anxiogène. Mieux vaut rassurer:
```suggestion
Un jury constitué des différentes équipes intéressés par les profils se réunira pour rencontrer les candidats avant de prendre une décision. Les modalités (visio, présentiel dans le respect des gestes de protection) en seront adaptées en fonction de la situation sanitaire et de celle des candidat·e·s.
```

Author: merci

Une fois la décision de l'équipe prise, c'est la Direction des services administratifs et financiers des services du Premier Ministre qui prend le relai pour constituer le dossier administratif. 

> Le délai de recrutement prend en moyenne 4 semaines. 

Reviewer: ```suggestion
Le délai de recrutement visé est de 4 semaines. 
```
